zerver/tests/test_auth_backends.py

```diff
@@ -684,7 +684,7 @@ def prepare_login_url_and_headers(self,
                                       subdomain: Optional[str]=None,
                                       mobile_flow_otp: Optional[str]=None,
                                       desktop_flow_otp: Optional[str]=None,
-                                      is_signup: Optional[str]=None,
+                                      is_signup: Optional[bool]=False,
```

Comment:
I think the type should be changed to `bool`?

Reply:
Thanks, changed it!